runelite-client/src/main/java/net/runelite/client/plugins/grandexchange/GrandExchangeSearchPanel.java

```diff
@@ -36,35 +39,64 @@
 import java.util.concurrent.ScheduledExecutorService;
 import javax.imageio.ImageIO;
 import javax.swing.ImageIcon;
-import javax.swing.JLabel;
 import javax.swing.JPanel;
+import javax.swing.JScrollPane;
 import javax.swing.SwingUtilities;
 import javax.swing.border.EmptyBorder;
 import lombok.extern.slf4j.Slf4j;
 import net.runelite.api.Client;
 import net.runelite.api.ItemComposition;
 import net.runelite.client.game.AsyncBufferedImage;
 import net.runelite.client.game.ItemManager;
+import net.runelite.client.ui.ColorScheme;
 import net.runelite.client.ui.components.IconTextField;
+import net.runelite.client.ui.components.PluginErrorPanel;
 import net.runelite.http.api.item.Item;
+import net.runelite.http.api.item.ItemClient;
 import net.runelite.http.api.item.ItemPrice;
 import net.runelite.http.api.item.SearchResult;
 
+/**
+ * This panel holds the search section of the Grand Exchange Plugin.
+ * It should display a search bar and either item results or a label (Searching..)
+ */
 @Slf4j
 class GrandExchangeSearchPanel extends JPanel
 {
-	private static final List<GrandExchangeItems> ITEMS_LIST = new ArrayList<>();
-
 	private final Client client;
 	private final ItemManager itemManager;
 	private final ScheduledExecutorService executor;
 
-	private ImageIcon search;
+	private ItemClient itemClient;
```

Comment:
Mark this final and initialize it here, preferrably make this constant

Reply:
What variable are you referring to?

Reply:
ItemClient